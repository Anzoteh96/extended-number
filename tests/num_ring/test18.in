441561832565242322789672349103343775425563555245367770
913330902365801198228086705392895260381728426503115879
925309440438458104611860641359270043638460712188965850
