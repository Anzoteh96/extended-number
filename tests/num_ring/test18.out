Enter the number base b:
Enter two numbers n and m:
b is 441561832565242322789672349103343775425563555245367770
n mod b is 30207237235316552648742007186207709530601316012380339
m mod b is 42185775307973459032515943152582492787333601698230310
n + m is 72393012543290011681257950338790202317934917710610649
n - m is 429583294492585416405898413136968992168831269559517799
n * m is 221717198054560328633990414615443487940243510359867240
n ** m is 56834255853197989998725329040997235462291785011362321
